client/game_state.py

```diff
@@ -17,6 +17,7 @@ def __init__(self, client, control_mode):
     def handle_state_data(self, data):
         """Handle game state data received from the server"""
         try:
+            # logger.debug("Received state data: " + str(data))
```

Comment:
In general, you shouldn't be commenting in/out log messages. You can control the log level you want at the logger level.